website/docs/cluster-management/managing-clusters-without-capi.mdx

```diff
@@ -182,18 +178,80 @@ kubectl create secret generic demo-01-kubeconfig \
 </TabItem>
 </Tabs>
 
-### Connect a cluster
+## Add a Cluster Bootstrap Config
 
-:::tip Before you start!
+This step ensures that Flux gets installed into your cluster. Create a cluster bootstrap config as follows:
 
-Make sure you've
+```bash
```

Comment:
So this step creates a secret not a cluster bootstrap config as mentioned above. I think we should explain why this secret is needed and somehow "link" it to the cluster bootstrap config.

Reply:
The previous step also creates a secret for the secret for the generated kubeconfig. Is the step you're referencing creating another secret, or do we have multiple doc sections about creating a secret?